core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message.properties

```diff
@@ -1,9 +1,9 @@
 pleaseRekeyAsap=\
-    Because of <a href="https://jenkins.io/security/advisory/2013-01-04/" target="_blank">a security vulnerability</a> discovered earlier, we need to \
+    Because of <a href="https://jenkins.io/redirect/rekey">a security vulnerability</a> discovered earlier, we need to \
     change the encryption key used to protect secrets in your configuration files on the disk. \
     This process scans a large portion of your <code>$JENKINS_HOME</code> (<code>{0}</code>), \
     find encrypted data, re-key them, which will take some time. \
-    See <a href="https://jenkins.io/security/advisory/2013-01-04/" target="_blank">this document</a> for more implications about different ways of doing this \
```

Comment:
Nothing against setting the target, by the way -- just the URL should stay the same. I'm sorry for the misleading comment asking for a revert.